Rakefile

```diff
@@ -108,6 +126,11 @@ def execute(cmd)
   end
 end
 
+def get_config_file
+  config_file = File.read "F:/Git/azure-sdk-for-ruby/config.json"
```

Comment:
minor: let's put relative path here

Reply:
yes, definitely! As I do my next round of updates, will get this updated.